aasemble/django/apps/api/v1/serializers.py

```diff
@@ -125,7 +125,7 @@ def update(self, instance, validated_data):
 
             class Meta:
                 model = mirrorsvc_models.Snapshot
-                fields = ('self', 'timestamp', 'mirrorset')
+                fields = ('self', 'timestamp', 'mirrorset', 'visible_to_v1_api')
```

Comment:
Sorry, I must have missed this when I reviewed it earlier. Why should this be part of the public API? I was thinking of it as an internal bit of book keeping.